03_DeviceSelectionAndSharedSources/app_resources/pplnLayoutMergeTest/shader_3.comp.hlsl

```diff
@@ -0,0 +1,15 @@
+#pragma wave shader_stage(compute)
+#include "common.hlsl"
+
+[[vk::binding(0,0)]] RWStructuredBuffer<SomeType> asdf;
+[[vk::binding(1,0)]] RWStructuredBuffer<SomeType> fdsa;
+[[vk::binding(2,0)]] RWByteAddressBuffer output;
```

Comment:
`shader_1` and `shader_3` define this as a runtime descriptor array and 1 sized

Check if its legal to bind a descriptor with layout that has a Runtime Array binding flag to a SPIR-V shader that doesn't decorate the binding like that.

(in simpler words, can you use a descriptor layout with runtime array at binding 2 with `shader_3`)